meshtastic/node.py

```
@@ -998,27 +1058,99 @@ def setURL(self, url: str, addOnly: bool = False) -> None:
+                    rollback_channel = original_channels_by_index.get(index)
+                    if rollback_channel is None:
+                        continue
+                    rollback_admin = admin_pb2.AdminMessage()
+                    rollback_admin.set_channel.CopyFrom(rollback_channel)
+                    try:
+                        self._send_admin(
+                            rollback_admin,
+                            adminIndex=admin_index_for_write,
+                        )
+                    # Best-effort rollback path; keep attempting remaining steps.
+                    except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)